build/Targets/Settings.props

```diff
@@ -167,6 +174,7 @@
         <RemoveIntegerChecks>true</RemoveIntegerChecks>
         <RootNamespace></RootNamespace>
         <MSBuildTargetsLanguageName>VisualBasic</MSBuildTargetsLanguageName>
+        <NoWarn>$(NoWarn);40057</NoWarn>
```

Comment:
I see this previously existed in some places, now we're applying it everywhere. Why? Can we add a comment for what this is working around?